endpoints/getting-started/clients/google-jwt-client.py

```diff
@@ -69,8 +69,7 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
     }
     response = requests.get(url, headers=headers)
 
-    response.raise_for_status()
-    return response.text
+    print(response.text)
 # [END endpoints_jwt_request]
```

Comment:
@TAOXUY Thank you! Could you also add a `response.raise_for_status()`after the `print()`?